assets/js/briqpay-order-meta-box.js

```diff
@@ -7,8 +7,51 @@ jQuery(function ($) {
 			briqpayAdmin.wrapperElement.slideToggle( "slow" );
 		},
 
+		 getUrlParameter : function getUrlParameter(sParam) {
+			var sPageURL = decodeURIComponent(window.location.search.substring(1)),
+				sURLVariables = sPageURL.split('&'),
+				sParameterName,
+				i;
+
+			for (i = 0; i < sURLVariables.length; i++) {
+				sParameterName = sURLVariables[i].split('=');
+
+				if (sParameterName[0] === sParam) {
+					return sParameterName[1] === undefined ? true : sParameterName[1];
+				}
+			}
+		},
+
 		init: function () {
 			$('body').on('click', '#briqpay_show_rules', briqpayAdmin.toggleRules );
+
+			if ($(".sync-btn-briqpay")) {
+				var btn = document.querySelector('.sync-btn-briqpay');
+
+				btn.addEventListener('click', function(e) {
+					e.preventDefault();
+					$.ajax({
+						type: 'POST',
+						data: {
+							id: briqpayAdmin.getUrlParameter('post'),
+							nonce: briqpayParamsMeta.update_order_orm_url_nonce,
+						},
+						dataType: 'json',
+						url: briqpayParamsMeta.update_order_orm,
+						success: function (data) {
+							if(data.success) {
+								window.location.reload();
+							}
+						},
+						error: function (data) {
+							console.log(data);
+						},
+						complete: function (data) {
+
+						}
+					});
+				});
```

Comment:
Can this be broken out into its own function? To keep the init function clean. You can register the listener here, but don't have an anonymous function, rather declare a function for this.